starburst-presto/presto.sh

```diff
@@ -35,18 +35,21 @@ function err() {
   return 1
 }
 
+# Purpose: Determine if presto is running or not
+function test_presto_with_query(){
+        # [ $(presto --server=localhost:${HTTP_PORT} --execute='select * from system.runtime.nodes;' | grep -q ${PRESTO_VERSION} && echo 0 || echo 1) -eq 0 ]
+        [ $(presto --server=localhost:${HTTP_PORT} --execute='select 1' | grep -q "1" && echo 0 || echo 1) -eq 0 ]
+}
+
 function wait_for_presto_cluster_ready() {
   # wait up to 120s for presto being able to run query
   for ((i = 0; i < 12; i++)); do
-    if presto --execute='select * from system.runtime.nodes;'; then
-      return 0
-    fi
+    test_presto_with_query && return 0
```

Comment:
I think the only real difference between the previous version and the current one is that now you provide the port.

So what about:

```suggestion
    if presto --server="localhost:${HTTP_PORT}" --execute="select 1; then
      return 0
    fi
```

Reply:
Cool!, that works as well :) 
Made new commit https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/pull/545/commits/5658bc26283670b3560235c0f67f8e0b2b535ea1